Multisig/Logic/Models/Chain.swift

```diff
@@ -400,7 +400,7 @@ extension Chain {
         case eip1271 = "EIP1271"
         case eip1559 = "EIP1559"
         case erc721 = "ERC721"
-        case relay = "RELAYING"
+        case relayingMobile = "RELAYING_MOBILE"
```

Comment:
nitpick: maybe keep the previous name? 🤔 From the mobile perspective we don't need to know that this feature is "mobile specific".

Reply:
I want the flag to have the same name. So if I am looking for it with curl, I am not confused.